### ISSUE TITLE
Error using sample smart quote function

### ISSUE THREAD

**Issue by vr8hub** [PowerShell/PSReadLine#569]
Powershell 5.1 on Win10 1709, using PSReadline 1.2.

I copied the sample PSReadline profile from here verbatim into an empty profile (Documents/WindowsPowerShell/profile.ps1). Everything seems to work except the smart quote. When I type a quote (single or double) when no text is selected, e.g. either on an empty line or at the end of some text, I get this:
An exception occurred in custom key handler, see $error for more information: Unable to find type [StringToken].

$error contains:
Unable to find type [StringToken].
At C:\Users\vrice\Documents\WindowsPowerShell\profile.ps1:177 char:20
+     if ($token -is [StringToken] -and $token.Kind -ne [TokenKind]::Ge ...
+                    ~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (StringToken:TypeName) [], RuntimeException
    + FullyQualifiedErrorId : TypeNotFound

I searched around the web trying to see if I could find something obvious, but I'm not sufficiently advanced in PowerShell to interpret what I found.

I don't get this when typing a brace {, I get the expected {} with the cursor between. But, that function doesn't refer to [StringToken].

I deleted the profile and cut-and-pasted it from here again, same result. (The site says the sample file is included with 1.2, but I couldn't find it anywhere, i.e. in or out of the PSReadline module subdirectory).

What am I missing?

**Comment by lzybkr** [comment]
Can you paste the code from your profile? Or copy again from https://github.com/lzybkr/PSReadLine/blob/9d592b875ebea83d6385b9cc6b6899584b08277a/PSReadLine/SamplePSReadlineProfile.ps1#L122 (though this version has some quirks too - but it's just a sample :)

There was a bug like this in that example long ago, but it was fixed.

**Comment by vr8hub** [comment]
I pasted the entire function here and was about to post it, but thought I'd try one more time. I cut-and-pasted the function from the [sample](https://github.com/lzybkr/PSReadLine/blob/master/PSReadLine/SamplePSReadlineProfile.ps1) again, but got the same result.

I was pretty convinced it was my error, so I kept looking, and doh! I somehow missed or deleted the two namespace lines at the very top! After putting those back, everything works fine.

My apologies for the noise! I suspected it was something obvious, I just couldn't figure out what it was.
